ledger-tool/src/main.rs

```diff
@@ -3339,9 +3368,21 @@ fn main() {
                 let print_account_contents = !arg_matches.is_present("no_account_contents");
                 if print_account_contents {
                     let print_account_data = !arg_matches.is_present("no_account_data");
+                    let print_encoding_format = match arg_matches.value_of("encoding") {
```

Comment:
Since you're adding this here, can you please add the `encoding` clap Arg to the `accounts` Subcommand?

Reply:
This should be resolved now